lib/prompts/createTemplatePrompt.ts

```diff
@@ -15,7 +15,7 @@ const templateTypeChoices: PromptChoices = [
 ] as const;
 
 interface CreateTemplatePromptResponse {
-  templateType: typeof templateTypeChoices[number]['value'];
+  templateType: typeof templateTypeChoices[number];
```

Comment:
This fixes the error for now, but it's not actually correct typing, since the prompt returns the value string rather than the object. Once we convert files that use this prompt, we'll run into problems.

I think we can use the new Typescript `satisfies` operator to fix this https://www.typescriptlang.org/docs/handbook/release-notes/typescript-4-9.html